src/config/generate-config.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-cm
+data:
+  GLUU_SECRET_ADAPTER: "kubernetes"
+  ADMIN_PW: "admin_Abcd1234#"
+  EMAIL: "support@gluu.org"
+  DOMAIN: "eoepca-dev.gluu.org"
+  ORG_NAME: "Deimos"
+  COUNTRY_CODE: "PT"
+  STATE: "NA"
+  CITY: "Lisbon"
+  GLUU_CONFIG_ADAPTER: "kubernetes"
+  LDAP_TYPE: "opendj"
```

Comment:
Although I understand these are just the "dummy" credentials, we should have a way of giving them values for a real production platform. Environment variables maybe? Or maybe document the location so an operator can edit it

Reply:
Moved variables to the setupCluster, and changed these values to placeholder variables. When installing, setupCluster changes these with sed to the real values to perform the installation, but the real values are not stored in these configuration files, only in the setupCluster.sh script.
Changed README to indicate where the values can be edited.